src/configuration/services/configuration.service.ts

```diff
@@ -10,10 +10,11 @@ import {
 } from '../models/configuration-item';
 import { generateSchema } from '@anatine/zod-openapi';
 import { ZodType, ZodTypeAny } from 'zod';
+import { SchemaObject } from 'openapi3-ts/oas31';
 
 interface ConfigurationEntryDescription {
   key: string;
-  schema: string;
+  schema: SchemaObject;
```

Comment:
???

Reply:
Yeah, for some reason...

Reply:
still makes no sense?

Reply:
Apprarently, the `@anatine/zod-openapi` introduced a change in the return type of the `generateSchema()` function; yarn installed an older version (peer dependency), which returned a string; the newer version (installed by pnpm) returns `SchemaObject`, thus the change.

Reply:
where are we using @anatine/zod-openapi?

Reply:
In the Configuration module; it's used to describe configuration options' schema. It converts zod schema to the openapi standard.

Reply:
you know... this whole time I read openapi as openai thinking you had somehow gotten into ai unexpectedly. Damn.